projects/fsastorefrontlib/src/cms-components/agent/agent-search-list/agent-search-list.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="searchResults$ | async as results; else loading">
-  <div class="mt-5">
+  <div class="mt-5" *ngIf="results.agents.length > 0; else noSearchResults">
```

Comment:
Agreed with Ana PO to add content in the case where there is no search results.

Reply:
Please add safe operators: results?.agents?.length

Reply:
> Please add safe operators: results?.agents?.length

Should I? Results is checked in above container if, and agents always exists it is only question if agents array is empty or have some data.